kore/test/Test/Kore/Variables/Fresh.hs

```diff
@@ -55,12 +74,213 @@ test_refreshVariable =
         in do
             assertRefreshes original metaVariableDifferentSort
             assertRefreshes metaVariableDifferentSort original
+
+    -- Target Variable
```

Comment:
Instead of the comments, could we put the tests for each combination of types in a `testGroup` so that it is visible in the output of a failing test?